static/styles/left-sidebar.scss

```diff
@@ -84,9 +85,9 @@ li.show-more-topics {
 
 #stream_filters {
     overflow: visible;
-    /* The -1px here prevents the scrollbar from going above the top of the container */
-    margin-top: -1px;
```

Comment:
This was a fix for the scrollbar moving out of the container. I tried testing, but couldn't reproduce it so I think it shouldn't be an issue. It's worth some thorough testing though, to ensure we don't break anything with this PR.

Reply:
yes, @vinitS101 I testing that... it's visually is not making any break